packages/tools/bcm2835-bootloader/scripts/update.sh

```diff
@@ -32,6 +32,7 @@
 
   cp -p $SYSTEM_ROOT/usr/share/bootloader/*.dtb $BOOT_ROOT
   cp -pR $SYSTEM_ROOT/usr/share/bootloader/overlays $BOOT_ROOT
+  cp -p $(get_build_dir linux)/arch/arm/boot/dts/overlays/ir-gpio-overlay.dtb $BOOT_ROOT/overlays/
```

Comment:
This function (and directory) do not exist on a running system. This line isn't even needed all together.